storage/src/main/java/org/apache/kafka/storage/internals/log/LogConfig.java

```diff
@@ -113,11 +113,13 @@ public String topicWarningMessage(String topicName) {
     public static class RemoteLogConfig {
 
         public final boolean remoteStorageEnable;
+        public final String remoteLogDisablePolicy;
```

Comment:
We might need a method to access this value. How about adding that as well to this pr ?

```
public String remoteLogDisablePolicy() {
        return remoteLogConfig.remoteLogDisablePolicy;
    }
```